be/src/storage/backgroud_task.h

```diff
@@ -0,0 +1,29 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
```

Comment:
`// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.`

Reply:
ok